docs/databases/elasticsearch/webserver-logs-with-elk-stack.md

```diff
@@ -248,54 +248,54 @@ To confirm the pipeline is working correctly, issue some requests to your webser
+- Scroll down to find "Pie chart" and click on it to start creating the chart.
+- Select "From a new search" to create a graph from a generic search query.
+- You'll see a large pie chart that is simply displaying the document count for the selected time period.
+- Click "Split Slices" on the left sidebar.
+- In the "Aggregation" dropdown, select "Terms."
+- Under the "Field" section, select `agent.raw`.
+- Change the "Size" option to 10.
+- Click on the green play button on the left sidebar.
 
 You should see a pie chart similar to the one below:
 
 ![Pie chart displaying most common user agents](/docs/assets/elk-kibana-pie-chart.png)
 
-This simple example just summarizes the dummy user agent strings we set in the `curl` command. In a real-world scenario, this same type of aggregation could be used to visualize the values in the `response` field to see how often 500 or 404 errors occur.
+This simple example summarizes the dummy user-agent strings we set in the `curl` command. In a real-world scenario, this same type of aggregation could be used to visualize the values in the `response` field to see how often 500 or 404 errors occur.
 
-To get a feel for Kibana, try creating a similar graph, but using a bar chart this time. The X axis should be a date histogram, and you can split the bars by adding sub-buckets to aggregate on the `agent.raw` field. Explore different possibilities!
+To get a feel for Kibana, try creating a similar graph, but use a bar chart this time. The X-axis should be a date histogram, and you can split the bars by adding sub-buckets to aggregate on the `agent.raw` field. Explore different possibilities!
 
 ## Further Reading
 
-From here on out, the [official Kibana documentation](https://www.elastic.co/guide/en/kibana/current/index.html) can help you learn how to graph log to create charts, graphs, and lots of other helpful visualizations. Remember that the `filter` section of the Logstash configuration file we wrote includes a `grok` directive to parse log fields, so you now have the ability to do things like:
+From here on out, the [official Kibana documentation](https://www.elastic.co/guide/en/kibana/current/index.html) can help you learn how to graph logs to create charts, graphs, and other helpful visualizations. Remember that the `filter` section of the Logstash configuration file we wrote includes a `grok` directive to parse log fields, so you now have the ability to do things like:
 
-- Average or sum of total bytes sent over the web server over a period of time
-- An aggregation (ranking of occurrences) of the top response code sent by your webserver (i.e. 200, 404, etc.)
-- Fast freetext search across all your logs using Elasticsearch
+- Calculate the average or sum of total bytes sent over the web server over a period of time.
+- Aggregate (rank occurrences) of the top response code sent by your webserver (i.e., 200, 404, etc.).
+- Search with fast, freetext across all your logs using Elasticsearch.
```

Comment:
"Perform fast freetext search across ..." would probably be the most accurate here.